kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/OperatorMain.java

```diff
@@ -43,6 +44,7 @@ public class OperatorMain {
     private static final Logger LOGGER = LoggerFactory.getLogger(OperatorMain.class);
     private static final String BIND_ADDRESS_VAR_NAME = "BIND_ADDRESS";
     private static final int DEFAULT_MANAGEMENT_PORT = 8080;
+    static final String HTTP_PATH_HEALTHY = "/healthy";
```

Comment:
In the proxy we went with `/livez`, following the convention and naming of some k8s components where they use distinct liveness and readiness endpoint.

Reply:
I think the operator and the proxy should be consistent (and any other containers we add).